com.google.AndroidStudioCanary.appdata.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>com.google.AndroidStudio.desktop</id>
```

Comment:
```suggestion
	<id>com.google.AndroidStudio</id>
```
the .desktop suffix here is not needed

Reply:
Thanks! Resolved. I wonder why it's present from the regular [Android Studio repo](https://github.com/flathub/com.google.AndroidStudio/blob/master/com.google.AndroidStudio.appdata.xml#L3)? This repo is directly forked from that.

Reply:
People tend to confuse the appstream ID/Application ID & the desktop file name.